doc/source/changes/version_0_25_1.rst.inc

```diff
@@ -19,6 +19,11 @@ Fixes
 
 * fixed displaying empty array when starting the viewer or a new session in it.
 
+* fixed default import of `view`, `edit` and `compare` functions in the viewer.
```

Comment:
Our users wouldn't understand that I think. I would write:

    fixed the `view()`, `edit()` and `compare()` functions not being available in the viewer console.

At least the word console needs to be in there